projects/askar.md

```diff
@@ -0,0 +1,90 @@
+4. Support for Decentralized Identity: As a secure storage solution tailored to decentralized identity systems, Askar plays a critical role in enabling decentralized, user-controlled identity management. It ensures that the cryptographic materials required for decentralized identities are stored securely and can be accessed reliably by agents and wallets.
+
+## Code of Conduct
+
+Askar follows the[ Hyperledger Code of Conduct](https://github.com/hyperledger/aries-askar/blob/main/CODE_OF_CONDUCT.md), which is what the OWF code of conduct is based on.  
+
+## TAC Sponsor
+
+* Tracy Kuhrt
+* TBD
+
+## Project License
+
+The project has an Apache 2.0 and MIT License:  [Askar - Apache License](https://github.com/hyperledger/aries-askar/blob/main/APACHE_LICENSE) and [Askar - MIT License](https://github.com/hyperledger/aries-askar/blob/main/MIT_LICENSE)
+
+## Source Control
+
+The project is hosted in GitHub and includes the following repositories:
+
+* [Askar](https://github.com/hyperledger/aries-askar) — code and documentation for Askar
+
+## Issue Tracker
+
+The issues are tracked using GitHub's Issues feature in the corresponding repository.
+
+## External Dependencies
+
+The ACA-Py dependency list is maintained in the [source repository](https://github.com/hyperledger/aries-askar/blob/main/Cargo.toml).
+
+## Release Methodology
+
+All proposed repositories have continuous deployment/delivery pipelines built using [GitHub Actions](https://github.com/features/actions). The individual packages follow the [semantic versioning](https://semver.org/) method, ensuring consistency and safety. Whenever a pull request is merged into the main branch that doesn't cause a major, minor or patch version increase, an alpha version is released.
+
+## Initial Maintainers
+
+| Name              | Github           | Organization                             |
+| ----------------- | ---------------- | ---------------------------------------- |
+| Andrew Whitehead  | andrewwhitehead  | Funded by Government of British Columbia |
+| Wade Barnes       | WadeBarnes       | Funded by Government of British Columbia |
+| Berend Sliedrecht | berendsliedrecht | Animo Solutions                          |
+| Timo Glastra      | TimoGlastra      | Animo Solutions                          |
+
+## Proposed Project Governance
+
+The current governance model under Hyperledger is consensus-based. This means that decisions are made through discussions, with the aim of community consensus, as outlined in the[ Aries Project Charter](https://docs.google.com/document/d/1F6RbR7xDaBt5CDJhqLJzR4c1pDJtyPGshp9fy6eVtSM/edit?usp=sharing). In cases where no clear consensus is established, a project Technical Steering Committee, or the maintainers (those with escalated GitHub privileges) are granted a louder voice. This approach has proven effective.
```

Comment:
Is the intention for this project to be distinct from the other Aries proposals and their governance processes when it moves to OWF?

Reply:
Askar is proposed to be a separate project at OWF, independent of the other artifacts stemming from the existing Hyperledger Aries project, with separate governance. That reflects the distinct purpose of the project, and community around it.